### PR TITLE
Expose translateToDevtools so data can be used without saving to disk

### PR DESCRIPTION
**Pull request by IhsanMujdeci**
Expose translateToDevtools so data can be used without saving to disk.
This is useful if anybody wants to do anything else with that data like save to s3 or pump into an event stream

### CODE REVIEW

File: src/write.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import * as pify from 'pify';
 import {DevToolsProfileNode} from './index';
 
 const writeFilep =
-    (pify(fs.writeFile) as (filename: string, data: {}) => Promise<void>);
+    (pify.default(fs.writeFile) as (filename: string, data: {}) => Promise<void>);
```
Comment (kenashcraft) [inline src/write.ts]: Looks like CI is complaining about this addition...

**Comment by kenashcraft** [comment]
The changes to package-lock.json seem unrelated to this change.  I think you should revert that file.

**Comment by kenashcraft** [comment]
I'm not a maintainer, but I would suggest adding a version of `write()` that takes a function with args `(filename: string, data: {})`.  By default, that function is `pify(fs.writeFile)`, but then you could provide your implementation that writes to S3 or wherever.

**Comment by ofrobots** [comment]
Hi @IhsanMujdeci. Thanks a lot for your contribution. It would great if you can make the changes that @kenashcraft suggested.

Also, we need you to sign this CLA before I can accept the contribution: https://cla.developers.google.com/. Sorry for the inconvenience and thanks in advance!

**Comment by IhsanMujdeci** [comment]
Thanks guys, ill make the changes soon. Thanks for even looking at the changes :)

File: src/write.ts (hunk 1 above)
Comment (IhsanMujdeci) [reply src/write.ts]: yeah very weird. My machine didn't like it unless i used `pify.default` ill make sure its working on ci and my machine

**Comment by mmarchini** [comment]
@IhsanMujdeci thank you for working on this! Did you had time to look into the suggested changes? I'm looking into saving the profile directly into S3 without going to disk first, and this PR would be really helpful for that :)

**Comment by ofrobots** [comment]
@mmarchini @IhsanMujdeci I decided to take a different approach in #34. Let me know if that approach is sufficient to address your use-cases.

**Comment by ofrobots** [comment]
Superceded by #34

**Pull request closed without merging**